src/hooks/useHover/__stories__/useHover.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import useHover from "../useHover";
+import UsageGuidelines from "../../../storybook/components/usage-guidelines/usage-guidelines";
+import "../../__stories__/general-hooks-stories.scss";
+
+<Meta title="Hooks/useHover" />
+
+# useHover
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Attaches a listener for <code>mouseover</code> and <code>mouseout</code> DOM events on a specific element, returning <code>isHovered</code> state.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const [hoverRef, isHovered] = useHover();
+      return (
+        <div ref={hoverRef} className="hooks-reference-element">
+          Hover me! {isHovered ? "Boom!" : undefined}
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    "Use this hook to get element hovered state",
+    "No arguments are needed just assign returned ref to the element"
+  ]}
+/>
+
+## Returns
+
+<FunctionArguments>
+  <FunctionArgument
+    name="ref"
+    type="React.MutableRefObject"
+    description={
+      <>
+        A React{" "}
```

Comment:
let's change to: React ref object to assign to the element which we want to track its hover state

and set the link over react ref.

by the way the link is cool!